lib/quby/questionnaires/entities/questions/date_question.rb

```diff
@@ -5,7 +5,7 @@ module Questions
         class DateQuestion < Question
           POSSIBLE_COMPONENTS = %i( day month year hour minute )
           COMPONENT_KEYS = Hash[POSSIBLE_COMPONENTS.zip %w( dd mm yyyy hh ii)]
-          COMPONENT_PLACEHOLDERS = Hash[POSSIBLE_COMPONENTS.zip %w( DD MM YYYY hh mm)]
+          COMPONENT_PLACEHOLDERS = Hash[POSSIBLE_COMPONENTS.zip %w( DD MM JJJJ uu mm)]
```

Comment:
Gezien we daadwerkelijk langzaam naar een internationale app aan het werken zijn. Is het een idee dit engels te laten en in de date_inputs view i18n te gebruiken en een extra data attribute met daarin de default COMPONENT_KEYS value. 
En dan in de dateValuesWithDefaults die waarde te gebruiken?